lib/Event/EventOptionsTrait.php

```diff
@@ -19,14 +21,14 @@
     /**
      * This array is used as key value storage for the options.
      *
-     * @var array
+     * @var OptionsResolver
```

Comment:
If I understand your point: you cannot determine the options from the OptionsResolver, it is there to be configured by the subscribers, it doesn't contain any option data, so you can't "get the options" from the resolver.